### ISSUE TITLE
[NPC] In Westfall zone NPC missing

### ISSUE THREAD

**Issue by TheSale** [WoWManiaUK/Blackwing-Lair#145]
Not first time I bring up this.
NPC` s for starting area quests are missing, and I think for me only, because psygoa was next to me and said that npc are there.
Janitor concluded that it might be problem whit phasing, and that I was teleported from draenai starting zone, however this is now from my human mage tested on lvl 6 without quests and at lvl 10 with quests for that area.
![westfal npc](https://user-images.githubusercontent.com/20323863/49159146-19155200-f324-11e8-8323-b231f4f35182.png)

after completing quests and get quests for westfall zone
![westfall 2](https://user-images.githubusercontent.com/20323863/49159157-1f0b3300-f324-11e8-9b13-3391c8a0517f.png)

shaman first time trying to solve with gm
![screenshot_10](https://user-images.githubusercontent.com/20323863/49159487-ecae0580-f324-11e8-87bb-22fadaf416dd.png)

Shaman after finished all quests in bloodmyst island and traveled trough regular means of transportation same issue
![westfal shamy 2](https://user-images.githubusercontent.com/20323863/49159591-2a129300-f325-11e8-8ce2-817786845f41.png)

**Comment by wow-mania** [comment]
OK we need to know which npcs. The info should be part of the report. With examples from wowhead or similar.

**Comment by TheSale** [comment]
https://www.wowhead.com/npc=42308/lieutenant-horatio-laine

**Comment by Echinaceax** [comment]
GM said NPC is still missing for quest delivery on this quest 
i will check 

could be related to #4461 
questline is a little messed up so phasing could be the issue 

**more info**
When quest added by self for testing / completing the NPC is there but only on GM mode, http://cata.cavernoftime.com/npc=42558 

Also note conflicting link info on ID numbers https://www.wowhead.com/npc=42308/lieutenant-horatio-laine

**Comment by mfranca0009** [comment]
I just tested this on the PTR now without GM mode, I was able to get up to "Shakedown At the Saldean's" without any issue and both NPCs (42558 and 42308) worked fine. NPC 42558 only shows up and phases in when "Lou's Parting Thoughts" (Eavesdrop on thugs) is complete. You start the series/storyline by utilizing NPC 42308 first.

Is this still an issue on live realm? 

**UPDATE**

Tested running through the storyline that Lieutenant Horatio Laine is apart of and did not see any NPC phasing issues three times. Both NPC IDs 42308 and 42558 are working properly.
